fs/partition/fs_mbr.c

```diff
@@ -36,7 +36,12 @@
  ****************************************************************************/
 
 #define MBR_SIZE                   512
+
+#if defined(CONFIG_FS_LARGEFILE)
+#define MBR_LBA_TO_BLOCK(lba, blk) (((uint64_t) le32toh(lba) * 512 + (blk) - 1) / (blk))
```

Comment:
we can simplify to:
```suggestion
#define MBR_LBA_TO_BLOCK(lba, blk) (((blkcnt_t)le32toh(lba) * 512 + (blk) - 1) / (blk))
```

Reply:
Thanks for the advice. That is better.
May I re-create the patch?